src/components/Modal/Modal.tsx

```diff
@@ -209,12 +209,14 @@ export const Modal = forwardRef<HTMLDivElement, ModalProps>(function Modal(
     (e: EventTarget) => {
       if (e instanceof Element) {
         const modalElement = findAncestor(e, `.${wrapperClass}`)
-        if (!modalElement) return false
+        if (!modalElement) {
+          return !isOpen
+        }
         const isModalWrapper = modalElement.classList.contains(wrapperClass)
         const isThisModalWrapper = modalElement.classList.contains(
           `${wrapperClass}-${id}`
         )
-        return isModalWrapper && !isThisModalWrapper
+        return isModalWrapper && !isThisModalWrapper && !isOpen
```

Comment:
sanity checking myself here – this shouldn't be `(...) || !isOpen`, bc we want this fn to return true whenever we're not opened, preventing a click?

Reply:
oop yeah good catch. I'm just going to change this to return early